plugins/modules/aci_l3out_logical_node.py

```diff
@@ -243,6 +249,7 @@ def main():
         node_id=dict(type="int"),
         router_id=dict(type="str"),
         router_id_as_loopback=dict(type="str", default="yes", choices=["yes", "no"]),
+        loopback_address=dict(type="str", aliases=["loopback"]),
```

Comment:
Please add an example depicting the removal of a loopback address from the node.